pkg/health/health_pod.go

```diff
@@ -84,6 +84,20 @@ func getCorev1PodHealth(pod *corev1.Pod) (*HealthStatus, error) {
 				health = msg.Health
 				status = msg.Status
 				messages = append(messages, msg.Message)
+			} else if containerStatus.RestartCount > 2 && containerStatus.LastTerminationState.Terminated != nil {
+				lastRestarted := containerStatus.LastTerminationState.Terminated.FinishedAt.Time
+				status = HealthStatusCode(containerStatus.LastTerminationState.Terminated.Reason)
```

Comment:
I think status also needs to move into the if statements, if it lasted restarted > 8h then the restart count and last termination reason ar enot relevant